incubator/azuremonitor-containers/templates/omsagent-daemonset.yaml

```diff
@@ -34,8 +34,15 @@ spec:
        resources:
 {{ toYaml .Values.omsagent.resources.daemonset | indent 9 }}
        env:
+       {{- if ne .Values.omsagent.env.clusterId "<your_cluster_id>" }}
+       - name: AKS_RESOURCE_ID
+         value: {{ .Values.omsagent.env.clusterId | quote }}
+       - name: AKS_REGION
```

Comment:
better to check explicitly for region being present (just like clusterid)

Reply:
> better to check explicitly for region being present (just like clusterid)
> sure, will add that.

Reply:
> > better to check explicitly for region being present (just like clusterid)
> > sure, will add that.

>> fixed.